app/assets/stylesheets/crayons.scss

```diff
@@ -33,3 +32,5 @@
 @import 'components/tabs';
 @import 'components/tags';
 @import 'components/tooltips';
+
+@import 'config/generator';
```

Comment:
Moving this to the bottom of the file to avoid potential issues caused by not using `!important` anymore.